input/pagecontent/ch-mhd-1.md

```diff
@@ -40,147 +40,20 @@ The DocumentReference elements which are allowed to be updated are defined in An
 ##### Example
```

Comment:
We should add:

When a transaction is used, it shall conform to the CHMhd1UpdateDocumentMetadataTransactionRequest profile.